lib/evss/pciu/email_address_response.rb

```diff
@@ -5,12 +5,16 @@
 module EVSS
   module PCIU
     class EmailAddressResponse < EVSS::Response
-      attribute :email_address, Hash
+      attribute :email_address, String
+      attribute :effective_date, String
```

Comment:
@hpjaj I changed this to be more like the `PhoneNumberResponse` and return the attributes instead of a hash that contains the attributes.

Reply:
👍